plugin.audio.bbcpodcasts/resources/lib/rssaddon/abstract_rss_addon.py

```diff
@@ -0,0 +1,277 @@
+from datetime import datetime
```

Comment:
Would it make sense to make a more generic module that you can then use in all your podcast/rss add-ons?

Reply:
the way I've done it here works for me. I use a git submodule, see https://github.com/Heckie75/kodi-submodule-rssaddon, which I use in 4 addons, i.e.
- https://github.com/Heckie75/kodi-addon-podcast
- https://github.com/Heckie75/kodi-addon-deutschlandfunk
- https://github.com/Heckie75/kodi-addon-bbc-podcasts
- https://github.com/Heckie75/kodi-addon-ard-aktuell

(some of them are not updated in the repo yet)

I guess that more generic modules must be good enough so that other developers can use them with ease. I'm not sure if this is the case at this moment.

At least there would have to be a kind of manual.

What do you think?

Reply:
Well, if 4 add-ons use it, I would say that a generic module should be worth it. But that is up to you, perhaps for the future? For now this looks good.